antsibull/build_acd_commands.py

```diff
@@ -162,15 +169,32 @@ def build_single_command():
         download_dir = os.path.join(tmp_dir, 'collections')
         os.mkdir(download_dir, mode=0o700)
 
+        # Download included collections
         included_versions = asyncio.run(download_collections(deps, app_ctx.galaxy_url,
                                                              download_dir,
                                                              app_ctx.extra['collection_cache']))
 
+        new_dependencies = DependencyFileData(
+            str(app_ctx.extra["acd_version"]),
+            str(ansible_base_version),
+            {collection: str(version) for collection, version in included_versions.items()})
```

Comment:
Not a request to change this code necessarily; when I wrote the type checking for `DependencyFileData.__init__()` I wondered if I should make it `t.Union[str, Version]` (where Version was either packaging Version or semantic version... I don't recall off the top of my head which one it is).

If you think it's more convenient to change the DependencyFileData API (now or in the future), that would be fine with me.